src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/DeploymentTests.ps1

```diff
@@ -69,6 +69,36 @@ function Test-NewDeploymentFromTemplateFile
 
 <#
 .SYNOPSIS
+Tests nested errors displayed when temployment put fails.
+#>
+function Test-NestedErrorsDisplayed
+{
+	# Setup
+	$rgname = Get-ResourceGroupName
+	$rname = Get-ResourceName
+	$rglocation = "EastUS"
+
+	try
+	{
+		# Test
```

Comment:
@vivsriaus Nice, error cases like this are really difficult, and I *love* seeing this test.